Airport_Tickets_System/ValidateUserInputOfFlightsUseCase.cs

```diff
@@ -0,0 +1,163 @@
+using System.Globalization;
+using System.Text;
+using Airport_Tickets_System.Utils;
+
+namespace Airport_Tickets_System;
+
+public class ValidateUserInputOfFlightsUseCase
```

Comment:
Classes should be nouns. 

```suggestion
public class FlightValidator
```